packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -455,7 +457,7 @@ export default connect(
   mapStateToProps,
   mapDispatchToProps
 )(
-  reduxForm({
```

Comment:
These type errors popped up after removing `react-scripts`. What's weird about them is that they seem valid—the prop types of `MonitorATMServicesViewImpl` don't actually match what's coming in from the HOC, so I am not sure why these were not failing before.